### DISCUSSION TITLE
Tutorial Outline Notebook

### DISCUSSION THREAD

**Discussion by jalew188** [MannLabs/alphabase#185] [Announcements]
This discussion shares what notebooks we already have using AlphaBase and AlphaRaw.

**Comment by jalew188** [comment]
1. Basic settings. https://github.com/MannLabs/alphabase/blob/main/nbs_tests/constants already contains notebooks describing the basic settings in alphabase, including atoms (atom.ipynb), isotopes (isotope.ipynb), amino acid (aa.ipynb) and modifications (modifications.ipynb)

2. Mass calculation. Peptide, isotopes, fragments.

3. Introduction of dataframe structure for peptides and their fragments.

**Comment by GeorgWa** [comment]
I have the following notebooks:
Alpharaw:
Get cycle time, injection times, CV, from a list of raw files for method development

AlphaBase:
Load MSfragger, Maxquant or DIANN PSMs and predict a library

Combine two libraries and repredict all properties with peptdeep.

**Reply by jalew188** [reply]
Thank you Georg!
